docs/developers-guide/drivers/plugins.md

```diff
@@ -67,7 +67,7 @@ The if-you're-interested reason is that Java's JDBC `DriverManager` won't use JD
 
 ## Building the driver
 
-To build a driver as a plugin JAR, check out the [Build-driver scripts README](https://github.com/metabase/metabase/tree/master/bin/build-drivers).
+To build a driver as a plugin JAR, check out the [Build-driver scripts README](https://github.com/metabase/metabase/tree/master/bin/build-drivers.sh).
```

Comment:
This is wrong, previously it pointed to a directory which had a `README.md` file, now it's pointing to a shell script. 

The name of this script did not change. That was `bin/build`.

The README file was moved to https://github.com/metabase/metabase/blob/master/bin/build-drivers.md .